src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -303,35 +304,31 @@ class MediaBaseViewer extends BaseViewer {
     }
 
     /**
-     * Determines if media should autoplay based on cached settings value.
+     * Handler for autoplay failure
+     * Overridden in child class
+     *
+     * @protected
+     */
+    handleAutoplayFail = () => {};
+
+    /**
+     * Autoplay the media
      *
      * @private
      * @emits volume
-     * @return {Promise}
+     * @return {void}
      */
-    autoplay() {
-        // Play may return a promise depending on browser support. This promise
-        // will resolve when playback starts. If it fails, we mute the video
-        // and try to play again.
-        // https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/
-        const autoPlayPromise = this.mediaEl.play();
-
-        if (autoPlayPromise && typeof autoPlayPromise.then === 'function') {
-            this.handleRate();
-            return autoPlayPromise
-                .then(() => {
-                    this.handleVolume();
-                })
-                .catch(() => {
-                    // Auto-play was prevented, try muted play
-                    this.setVolume(0);
-                    this.mediaEl.play();
-                });
-        }
-
-        // Fallback to traditional autoplay tag if play does not return a promise
-        this.mediaEl.autoplay = true;
-        return Promise.resolve();
+    async autoplay() {
```

Comment:
Await/async isn't currently used or supported in Preview, I'm afraid. You should be able to replace this with native promises pretty easily, though.